src/components/CreateVmWizard/CreateVmWizard.js

```diff
@@ -386,13 +398,16 @@ class CreateVmWizard extends React.Component {
     this.props.onCreate(basic, nics, disks, correlationId)
   }
 
-  wizardGoToStep (newStepIndex) {
+  wizardGoToStep (newStepIndex, moveBackward = false) {
```

Comment:
instead of dedicated flag we can compare indices i.e. `moveBackward = nextStepIndex < activeStepIndex`